autopush_rs/src/client.rs

```diff
@@ -422,6 +439,20 @@ where
                     return Ok(next_state.into());
                 }
                 match try_ready!(self.data.input_or_notif()) {
+                    Either::A(ClientMessage::BroadcastSubscribe { broadcasts }) => {
```

Comment:
nit: "broadcasts" is being used as a parameter, local variable and key here. It's a bit confusing.

Reply:
It's used to destructure, then its re-bound locally as the Vec, and passed into a function. I can remove an intermediary reassignment though and pass the resulting Vec<Service> directly to the single use of it in a function, that might clear it up some.